src/sys/windows/tcp.rs

```
@@ -365,6 +365,7 @@ impl TcpListener {
 
     pub fn accept(&self) -> io::Result<(TcpStream, SocketAddr)> {
         try_io!(self, accept).and_then(|(inner, addr)| {
+            self.io_blocked_reregister()?;
```

Maybe we should ignore the error here and return the stream even if we error?

---

Makes sense, will ignore result.